test/lua/module/pandoc-path.lua

```diff
@@ -0,0 +1,17 @@
+local tasty = require 'tasty'
+local path = require 'pandoc.path'
+
```

Comment:
Wouldn't hurt to add a few more tests here.

Reply:
I could copy the tests from hslua-module-path, but that seems redundant. Would checking for the existence of all documented functions be a good solution?

Reply:
agreed.